en/docs/develop/authenticating-and-authorizing-rest-apis.md

```diff
@@ -61,46 +50,7 @@ Let's learn how to authenticate and authorize REST APIs:
             http_method = "all"
             permissions = ["p1","p2"]
             ```
-
-3.  To configure intermediate certificate validation, configure the
-    following in the `           deployment.toml          ` file as given
-    below.
-
-    <table>
-    <thead>
-    <tr class="header">
-    <th>Parameter</th>
-    <th>Description</th>
-    <th>Sample Value</th>
-    </tr>
-    </thead>
-    <tbody>    
-    <tr class="even">
-    <td><strong>cert_cns</strong></td>
-    <td>This specifies the context paths of the intermediate certificates.</td>
-    <td><code>               localhost              </code></td>
-    </tr>
-    <tr class="odd">
-    <td><strong>exempt</strong></td>
-    <td>This specifies the context paths that needs to be excempted from intermediate certificate validation.</td>
-    <td><br />
-    </td>
-    </tr>
-    </tbody>
-    </table>
-
-    !!! example
```

Comment:
Please add config for anabling the intermediate certificate validation

Reply:
This has not been templated yet. Reported in 
https://github.com/wso2/product-is/issues/6779.